app/code/Magento/Checkout/view/frontend/templates/cart/coupon.phtml

```diff
@@ -7,7 +7,7 @@
 ?>
 <div class="block discount"
      id="block-discount"
-     data-mage-init='{"collapsible":{"openedState": "active", "saveState": false}}'
+     data-mage-init='{"collapsible":{"active": <?= strlen($block->getCouponCode()) ? 'true' : 'false' ?>, "openedState": "active", "saveState": false}}'
```

Comment:
Hi @iGerchak. I think we can simply convert strlen($block->getCouponCode()) to the bool, instead of use comparation operator.

Reply:
I believe it was done just in the same way as on line 23 https://github.com/magento/magento2/pull/24467/files#diff-064408e40cf6c624ca3a7a8adfda18a8R23

In this case we should fix that part line also